tool/site/client-react/src/pages/sendPreSignPage.tsx

```diff
@@ -19,6 +22,33 @@ const SendPreSignPage: React.FC<SendPreSignPageProps> = ({
   const {baseUrl} = useContext(AppContext)
   const [sendConfirmed, setSendConfirmed] = useState<boolean>(false)
   const retrievePrescriptionTask = () => retrievePrescription(baseUrl, prescriptionId)
+
+  /* Pagination ------------------------------------------------ */
+  const [addedListener, setAddedListener] = useState(false)
+  const [cookies] = useCookies()
+  const LEFT_ARROW_KEY = 37
+  const RIGHT_ARROW_KEY = 39
+  const handleKeyDown = useCallback((e: any) => {
+    if (e.keyCode === LEFT_ARROW_KEY) {
+      const previousPrescriptionId = cookies["Previous-Prescription-Id"]
+      if (previousPrescriptionId) {
+        customWindow.location = `${baseUrl}prescribe/edit?prescription_id=${encodeURIComponent(previousPrescriptionId)}`
```

Comment:
Can we use `tool/site/client-react/src/browser/navigation.ts#redirect()` for redirects? That way we can mock it out for testing (js-dom doesn't support navigation)